modules/ROOT/pages/sync/auth.adoc

```diff
@@ -28,4 +39,11 @@ A secret will be created as a result of this binding.
 
 NOTE: The mount path must be set to `/opt/keycloak` for the {sync-service} to be configured to use the secret.
 
+== Making secret available
+
+By default secret is not visible to server application
```

Comment:
Minor grammar thing. Should probably be like this:

> By default, **the** secret is not visible to **the** server application

Reply:
+1